### PR TITLE
Issue/462/mock cluster ensemble

### PR DESCRIPTION
**Pull request by eduardojsbarroso**

### CODE REVIEW

File: examples/draw_M_z_from_hmf.ipynb [hunk 1]
```diff
@@ -0,0 +1,863 @@
...
+    "\n",
+    "    # Keeps drawing until N points are accepted  \n",
+    "    ran_logm = [] # output list of random numbers  \n",
+    "    ran_z = [] # output list of random numbers  \n",
+    "    while naccept < N:  \n",
+    "        # draw (logm,z) from uniform distribution\n",
+    "        # draw p from uniform distribution\n",
+    "        logm = np.random.uniform(logm_min,logm_max) # x'  \n",
+    "        z = np.random.uniform(zmin,zmax) # x'  \n",
+    "        p = np.random.uniform(pmin,pmax) # y'  \n",
```
Comment (eduardojsbarroso) [inline examples/draw_M_z_from_hmf.ipynb:348]: I changed the pdf to start from pmin instead of 0

File: examples/draw_M_z_from_hmf.ipynb [hunk 2]
```diff
@@ -0,0 +1,863 @@
...
+   "source": [
+    "clusterensemble.make_stacked_radial_profile(tan_component='DS_t', cross_component='DS_x')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "771c1382",
+   "metadata": {},
+   "source": [
+    "## Covariance (Bootstrap, sample, Jackknife) of the stack between radial bins\n",
```
Comment (eduardojsbarroso) [inline examples/draw_M_z_from_hmf.ipynb:706]: I don't know if we are keeping the covariance analysis on this notebook.

File: examples/draw_M_z_from_hmf.ipynb (hunk 1 above)
Comment (combet) [reply examples/draw_M_z_from_hmf.ipynb]: I reverted to 0 as the sanity check was not passing when using pmin. I remember having to convince myself that this was what needed to be used, but cannot think of the reason now. Definitely needs to go back to it, but the behaviour starting from 0 is the right one.

File: examples/draw_M_z_from_hmf.ipynb (hunk 2 above)
Comment (combet) [reply examples/draw_M_z_from_hmf.ipynb]: I've left it for the time being, but maybe we can just use one covariance and not compute the three. Let's see with whoever reviews the PR.

File: examples/draw_M_z_from_hmf.ipynb (hunk 1 above)
Comment (m-aguena) [reply examples/draw_M_z_from_hmf.ipynb]: If you sort a random number starting from `pmin` points around `P(m, z)=pmin` will never be selected. You want them selected `pmin/pmax` of the time.

**Review by m-aguena — APPROVED** [review]

**Pull request merged**